### ISSUE TITLE
Task :react-native-health-connect:compileDebugKotlin FAILED

### ISSUE THREAD

**Issue by nikolajevric** [matinzd/react-native-health-connect#5]
I have a question regarding the react-native-health-connect package/library. I have installed the package, but when I build my app, an error appears, indicating that there is a problem with the package.

I have been using this: "react-native": "0.64.3", java 15.0.1 versions of these programming languages in my project.
And i have also installed "react-native-health-connect": "0.1.0-alpha.3" package in my project.

I was wondering if you could help me with this package and if you are aware of the issue that is causing the error. Additionally, I was curious about the recommended versions of JVM and/or react-native to use in order to avoid this problem.
I would greatly appreciate any information or guidance you can provide on this matter.

Here is a example of an error that appears:

> Task :react-native-health-connect:compileDebugKotlin FAILED

FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':react-native-health-connect:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

And also a screenshot of an error:
![hc](https://user-images.githubusercontent.com/81028760/217832863-f1cbc59f-cb6b-4ff5-8805-53925761723e.JPG)

 Thank you for your time and expertise.

**Comment by matinzd** [comment]
Hi @nikolajevric 

Can you provide more detail on this? I can see there are more logs down there. Maybe use [pastebin](https://pastebin.com/) to send me the whole log. Thanks

**Comment by nikolajevric** [comment]
Yes of course! Here is the log of my error:
Pastebin: https://pastebin.com/MJMiLHy4
Picture:
![hb2](https://user-images.githubusercontent.com/81028760/217840828-ba6f3898-651e-4ce1-9343-5f3dda66a33c.JPG)

**Comment by matinzd** [comment]
> Yes of course! Here is the log of my error: Pastebin: https://pastebin.com/MJMiLHy4 Picture: ![hb2](https://user-images.githubusercontent.com/81028760/217840828-ba6f3898-651e-4ce1-9343-5f3dda66a33c.JPG)

What is the version of your JDK? `java --version`

**Comment by nikolajevric** [comment]
I encountered an issue with the library, but I was able to resolve it by changing the version of React Native and deleting the "// @ts-expect-error" line from "node_modules/react-native-health-connect/src/index.tsx". However, at present, the library supports only a limited number of record class types. I was wondering if there are plans to update the library in the near future to support all the record class types from the documentation that you have posted. Thank you for your help.
